catroid/src/androidTest/java/org/catrobat/catroid/test/content/actions/PenDownActionTest.java

```diff
@@ -170,6 +173,7 @@ public void testMultiplePenDownActions() {
 	private void createTestProject() {
 		Project project = new Project(ApplicationProvider.getApplicationContext(), projectName);
 		XstreamSerializer.getInstance().saveProject(project);
-		ProjectManager.getInstance().setCurrentProject(project);
+		final Lazy<ProjectManager> projectManager = inject(ProjectManager.class);
```

Comment:
Why do we sometimes use lazy and sometimes not?

Reply:
@juliajulie95 In most cases `Lazy` is used to save the reference in order to postpone the resolving to directly before access. In case there are multiple accesses in a succession, the reference is only resolved once to reduce the overhead.